third_party/move/tools/move-cli/src/base/test.rs

```diff
@@ -13,13 +13,14 @@ use move_compiler::{
     PASS_CFGIR,
 };
 use move_coverage::coverage_map::{output_map_to_file, CoverageMap};
-use move_package::{compilation::build_plan::BuildPlan, BuildConfig};
+// if windows
```

Comment:
This looks like an awkward place to have this, is it supposed to be conditional inputs?

Reply:
This was moved by autoformatter after reformating imports. Move it back to where it was, before the cfg!